src/customActions/connectWalletActions.ts

```diff
@@ -244,27 +291,18 @@ export const connectWalletActions =
           ...args,
           wallet
         } as sendBatchTransactionsWithConnectParameters),
-      writeContract: (args) =>
-        writeContract(client, {
-          ...args,
-          wallet
-        } as WriteContractWithConnectParameters),
+      writeContract: (args): any => 
+        writeContract(client, {...args, wallet } as any),
       signMessage: (args) =>
         signMessage(client, {
           ...args,
           wallet
         } as SignMessageWithConnectParameters),
-      verifyMessage: (args) =>
-        verifyMessage(client, {
-          ...args,
-          wallet,
-          apiKey
-        } as VerifyMessageWithConnectParameters),
       simulateContract: (args) =>
         simulateContract(client, {
           ...args,
           wallet
         } as SimulateContractWithConnectParameters)
     })
 
-export { getTransaction, sendBatchTransactions, sendTransaction, simulateContract, verifyMessage, writeContract }
```

Comment:
Why deleting the verifymessage? We need it to be able to verify signature when wallet is not deployed